CosmosContracts/juno-tools
# feat(pages): update escrow notice in register and fund page

Fixes #87

## Description

This partial PR updates escrow notice in airdrops register and fund page. If #90 is merged, will change base PR to `development`.

## Changes

List any technical changes.

- [x] remove register and fund sticky button behavior
- [x] update escrow notice in register and fund page

## Screenshots

<img width="1552" alt="Screen Shot 2022-03-11 at 04 21 19 AM" src="https://user-images.githubusercontent.com/8220954/157758918-419616fa-4147-4ae3-b265-f4e2702b3963.png">

<img width="1552" alt="Screen Shot 2022-03-11 at 04 22 36 AM" src="https://user-images.githubusercontent.com/8220954/157758944-760cc0c4-2dcd-480f-977f-3e90aeb17a0b.png">


## Testing Steps

As a reviewer, what steps should I take to verify this is working correctly?

- go to either register or fund page
- use an airdrop address which has not been escrowed

## Links

\-
